core/src/main/java/eu/fasten/core/data/Constants.java

```diff
@@ -0,0 +1,17 @@
+package eu.fasten.core.data;
+
+public class Constants {
+
+    public static final String mvnCoordinatePartsJoin = ":";
+
+    public static final String mavenForge = "mvn";
```

Comment:
You use `mvn` in other cases, but `maven` here. I think this constant should be called `mvnForge`

Reply:
Renamed it as well